File: spec/python/test_params_call_short.py
import unittest

from params_call_short import ParamsCallShort

class TestParamsCallShort(unittest.TestCase):
    def test_params_call_short(self):
        r = ParamsCallShort.from_file("src/term_strz.bin")

        self.assertEqual(r.buf1.body, "foo|b")
        self.assertEqual(r.buf2.body, "ar|ba")
        self.assertEqual(r.buf2.trailer, 0x7a)

File: spec/lua/test_params_call_short.lua
local luaunit = require("luaunit")

require("params_call_short")

TestParamsCallShort = {}

function TestParamsCallShort:test_params_call_short()
    local r = ParamsCallShort:from_file("src/term_strz.bin")

    luaunit.assertEquals(r.buf1.body, "foo|b")
    luaunit.assertEquals(r.buf2.body, "ar|ba")
    luaunit.assertEquals(r.buf2.trailer, 0x7a)
end
